app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -220,6 +220,15 @@ class AdvancedFilter extends React.Component {
             'This filter is based on "Options to Reduce Quarantine for Contacts of Persons with SARS-COV-2 Infection Using Symptom ' +
             'Monitoring and Diagnostic Testing" released by the CDC on December 2, 2020. For more specific information, see Appendix A in the User Guide.',
         },
+        {
+          name: 'sms-blocked',
+          title: 'SMS Blocked (Boolean)',
+          description: 'Monitorees that have blocked SMS communications with Sara Alert',
+          type: 'boolean',
+          tooltip:
```

Comment:
just wanted to confirm whether or not this language has been verified with the PO team

Reply:
It has! It comes directly from them.